dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_dashboard.feature

```diff
@@ -32,6 +33,15 @@ Feature: Using the teacher dashboard
     Then I navigate to teacher dashboard for the section I saved
     And I wait until element "#uitest-course-dropdown" is visible
     And I select the "All the Things! *" option in dropdown "uitest-course-dropdown"
+    Then I append "/?enableExperiments=section_progress_v2" to the URL
+
+    # Toggle to V2 progress view
+    Then I click selector "#ui-test-link-to-new-progress-view"
+    And I wait until element "h6:contains(Icon Key)" is visible
+    And I wait until element "#ui-test-progress-table-v2" is visible
+    And element "#ui-test-progress-table-v2" is visible
```

Comment:
```suggestion
```
I think this is covered by the line above?

Reply:
Correct... chagned!